File: Daily Test/Java/First X Vowels 19.4.2026.java
import java.util.*;

public class Hello {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int x = s.nextInt();
        String str = s.next();

        int count = 0;

        for(int i = 0; i < str.length(); i++) {
            char c = str.charAt(i);

            if("aeiouAEIOU".indexOf(c) != -1) {
                System.out.print(c);
                count++;
            }

            if(count == x) break;
        }
    }
}

File: Daily Test/C/First X Vowels 19.4.2026.c
#include <stdio.h>
#include <string.h>

int main() {
    int x, count = 0;
    char str[1001];

    scanf("%d", &x);
    scanf("%s", str);

    for(int i = 0; str[i] != '\0'; i++) {
        char c = str[i];

        if(strchr("aeiouAEIOU", c)) {
            printf("%c", c);
            count++;
        }

        if(count == x) break;
    }

    return 0;
}